### PR TITLE
Add --tx flag to control symbolic tx

### PR DESCRIPTION
**Pull request by mossberg**
Currently, the manticore ethereum cli uses coverage based logic for determining when to stop running symbolic transactions. This can be somewhat faulty currently due to bugs in coverage computation. It can be useful to manually be able to control the number of symbolic tx, for example, to limit unnecessary state exploration.

This adds a `--txlimit` flag to the CLI, whose argument must be a positive integer.

### CODE REVIEW

File: manticore/__main__.py [hunk 1]
```diff
@@ -58,6 +62,9 @@ def parse_arguments():
                               "(default mcore_?????)"))
     parser.add_argument('--version', action='version', version='Manticore 0.1.5',
                          help='Show program version information')
+    parser.add_argument('--tx', type=check_positive,
```
Comment (yan) [inline manticore/__main__.py]: `--tx` is an ambiguous name for this parameter; on first glance it looks like it represents data to transmit. Maybe `--max_transactions`, `--max_tx_count`, `--max_tx`, or `--tx_limit`?

Comment (mossberg) [reply manticore/__main__.py]: great point, for some reason i have a mild dislike of underscores in cli flags like that though lol
`--maxtx` `--txlimit` `--txmax`?

Comment (yan) [reply manticore/__main__.py]: Out of those, I like `--txlimit`

**Comment by mossberg** [comment]
the arg parse code doesn't check to see if `int(value)` has an exception, but it's kinda cool, since the function is named `positive`, argparse's default exception handler still makes sens

```
[I] mark ubuntu /m/h/c/m/e/evm (dev-tx-arg) ❯ manticore two_tx_ovf.sol  --txlimit zzz
usage: manticore [-h] [--coverage COVERAGE] [--data DATA] [--env ENV]
                 [--file FILES] [--policy POLICY] [--profile] [--procs PROCS]
                 [--timeout TIMEOUT] [-v] [--workspace WORKSPACE] [--version]
                 [--txlimit TXLIMIT]
                 argv [argv ...]
manticore: error: argument --txlimit: invalid positive value: 'zzz'

```

**Pull request merged**